template/base.html.jinja

```diff
@@ -232,11 +247,20 @@ Answer (including related code snippets if available and markdown links to relev
           appId: 'R39QY3MZC7',
           indexName: 'xrpl',
         },
-      }
+      },
+      support: {
+        disclaimer: {
+          displayedBeforeAnswer: false,
+        }
+      },
     },
   };
 </script>
-<script id="initjs" type="module" src="https://www.unpkg.com/@xrpl/ai-web/dist/init.js"></script>
+<script type="module" src="{{currentpage.prefix}}assets/js/xrplai.js"></script>
+{#
+<script id="initjs" type="module" src="{{currentpage.prefix}}assets/js/xrplai.js"></script> #}
```

Comment:
Are these necessary? Both seem to be importing the same code and this seems like this might be the same script as near the top of this file as well.

Reply:
waiting for the ai-js library to be updated and deployed on unpkg so I don't have to import those files

Reply:
Similar to the comment above, please push changes to this branch so it's ready to go as if the other half is already done. (Locally can be set up to allow you to test your code differently). 

I tend to think that PR's should pretend they're already production code, or be in draft mode, to clearly signal that as-written these changes shouldn't be merged. That way no accidents happen where code is fine "except one last change" and then gets merged.